components/fileselect/templates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Templates
+page_title: FileSelect - Templates
```

Comment:
The dash between the component name and the corresponding title is being used inconsistently throughout the Blazor docs. I suggest removing it for simplicity. 

```suggestion
page_title: FileSelect Templates
```